### ISSUE TITLE
`mu pipeline logs` gives `ResourceNotFoundException: The specified log group does not exist.` error

### ISSUE THREAD

**Issue by timbaileyjones** [stelligent/mu#443]
While testing another PR, I found that `mu pipeline logs` command displays information from the pipelines, but also shows this error:
```bash
$ mu pipeline logs
[... normal, expected output ...]
func1 ▶ ERROR  ResourceNotFoundException: The specified log group does not exist.
	status code: 400, request id: f7260741-7f69-4772-b4cc-7c6a9c22d264
```

This error **does not** occur with the `-f` (follow) option.

Note that everything in the pipeline & deployments is still actually working.  It would be nice to eliminate this spurious message.

**Comment by jay13jay** [comment]
Hey uh. Considering you guys fired me for having a heart attack, can you
maybe stop spamming me?

**Comment by jay13jay** [comment]
Yeah I know I can unsubscribe, but isn't it kinda funny you idiots didn't
think to do that?

Good luck in your future endeavors, dumb asses.

**Comment by jay13jay** [comment]
Some of y'all should consider looking for better companies. This one is
going nowhere, and considering they fire people for major medical issues I
think it's quite clear they give 0 shits about your well being

**Comment by jay13jay** [comment]
on that note, that legal document should be arriving soon :)
